airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -437,6 +437,9 @@ class DataprocCreateClusterOperator(BaseOperator):
     :type project_id: str
     :param region: leave as 'global', might become relevant in the future. (templated)
     :type region: str
+    :parm delete_on_error: If true the claster will be deleted if created with ERROR state. Default
+        value is true.
+    :type delete_on_error: bool
```

Comment:
should there be a similar configurable parameter like "delete_on_configuration_mismatch" that defaults to False?

Reply:
I think this case will be difficult to handle in all cases correctly. You will not always need to create a new cluster when the configuration is different eg. additional components installed do not affect the usability of the cluster.